client/src/components/WriteComment/WriteComment.test.tsx

```diff
@@ -28,7 +28,7 @@ describe('WriteComment smart component', () => {
         renderer = mount(<Provider store={store}><WriteComment {...props} /></Provider>);
     });
 
-    test('render-success', () => {
+    test.skip('render-success', () => {
```

Comment:
!

Reply:
There is pr in which it will be changed.

Reply:
I hope